src/commands/deploy/buildImageInAzure/tar.d.ts

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * The `@types/tar` package is riddled with inaccuracies, so it's easier
+ * to just declare typings for it here
+ */
+declare module "tar" {
```

Comment:
I'd include a comment with a link to note that this is originally from the Docker extension. Not worth the effort to make a package though, IMO.